src/components/EResourceSections/Identifier.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import {
+  Col,
+  KeyValue,
+} from '@folio/stripes/components';
+
+import { getResourceIdentifier } from '../utilities';
+
+const Identifier = (
```

Comment:
Can we call this something like `EResourceIdentifier` instead so it's obvious what this component is? And move it up a level out of `EResourceSections`?